modules/git/diff_test.go

```diff
@@ -30,7 +30,7 @@ index d8e4c92..19dc8ad 100644
 @@ -1,9 +1,10 @@
  --some comment
 --- some comment 5
-+--some coment 2
++--some comment 2
```

Comment:
From CI:
> --- FAIL: TestCutDiffAroundLine (0.00s)
2090    diff_test.go:139: 
2091        	Error Trace:	diff_test.go:139
2092        	Error:      	Not equal: 
2093        	            	expected: "diff --git a/aaa.sql b/aaa.sql\n--- a/aaa.sql\n+++ b/aaa.sql\n@@ -1,9 +1,10 @@\n --some comment\n--- some comment 5\n+--some coment 2"
2094        	            	actual  : "diff --git a/aaa.sql b/aaa.sql\n--- a/aaa.sql\n+++ b/aaa.sql\n@@ -1,9 +1,10 @@\n --some comment\n--- some comment 5\n+--some comment 2"

Reply:
@delvh so basically this is a legit test?

Reply:
Reverted in 399761b